st_package_reviewer/check/file/__init__.py

```diff
@@ -42,6 +43,18 @@ def file_context(self, path):
             pass
         return self.context("File: {}".format(path))
 
+    def visit_all_pyfiles(self):
+        pyfiles = self.glob("**/*.py")
+        for self.current_file in pyfiles:
+            with open(self.current_file, "r") as f:
+                try:
+                    root = ast.parse(f.read(), self.current_file)
```

Comment:
If it's cached, how would I access `self.current_file` in the visitor methods? Should I create a dict that maps files to ASTs, and walk over the dict instead?